elastic/kibana
# [maps] fix 'by value' map does not fill dashboard panel on initial page load in 8.10

Fixes https://github.com/elastic/kibana/issues/165183

### Test instructions
1) install any sample data set
2) open new dashboard
3) Use "Add panel" buttons to add by-value map with only base map layer
4) Ensure map tiles fill entire map panel in dashboard
